### DISCUSSION TITLE
Engineering mode x50

### DISCUSSION THREAD

**Discussion by shayeight** [xeon1989/Proton-X50-APK-Installer#7] [Ideas]
Do you know to enter engineering mode for x50 infotainment system? Look at for geely coolray is jh3280 to enter that mode.. any idea?

**Comment by xeon1989** [comment]
Both Geely Coolray and Proton X50 IHU are running on different system, therefore the code are different too. 
Mind to share what are you trying to achieve with Engineering Mode?
